drivers/hmis/app/models/hmis/unit.rb

```diff
@@ -22,6 +24,10 @@ class Hmis::Unit < Hmis::HmisBase
   has_many :unit_occupancies, class_name: 'Hmis::UnitOccupancy', inverse_of: :unit, dependent: :destroy
   has_many :active_unit_occupancies, -> { active }, class_name: 'Hmis::UnitOccupancy', inverse_of: :unit
   has_many :current_occupants, through: :active_unit_occupancies, class_name: 'Hmis::Hud::Enrollment', source: :enrollment
+  has_many :opportunities, as: :owner, class_name: 'Hmis::Ce::Opportunity', inverse_of: :owner
+  # This is the latest opportunity, which could be either active or closed with an accepted referral.
+  # This relies on the assumption that if there is an active opportunity, it should be the latest one.
+  has_one :latest_opportunity, -> { order(created_at: :desc) }, as: :owner, class_name: 'Hmis::Ce::Opportunity', inverse_of: :owner
```

Comment:
suggest adding id as tie breaker, maybe:
```suggestion
  has_one :latest_opportunity, -> { order(created_at: :desc, id: :desc) }, as: :owner, class_name: 'Hmis::Ce::Opportunity', inverse_of: :owner
```